web/modules/custom/hoeringsportal_citizen_proposal/modules/hoeringsportal_citizen_proposal_fixtures/src/Fixture/CitizenProposalFixture.php

```diff
@@ -0,0 +1,115 @@
+<?php
+
+namespace Drupal\hoeringsportal_citizen_proposal_fixtures\Fixture;
+
+use Drupal\content_fixtures\Fixture\AbstractFixture;
+use Drupal\content_fixtures\Fixture\DependentFixtureInterface;
+use Drupal\content_fixtures\Fixture\FixtureGroupInterface;
+use Drupal\Core\Datetime\DrupalDateTime;
+use Drupal\hoeringsportal_base_fixtures\Fixture\MediaFixture;
+use Drupal\hoeringsportal_base_fixtures\Fixture\ParagraphFixture;
+use Drupal\hoeringsportal_base_fixtures\Helper\Helper;
+use Drupal\node\Entity\Node;
+
+/**
+ * Page fixture.
+ *
+ * @package Drupal\hoeringsportal_citizen_proposal_fixtures\Fixture
+ */
+class CitizenProposalFixture extends AbstractFixture implements DependentFixtureInterface, FixtureGroupInterface {
+
+  /**
+   * The fixtures helper service.
+   *
+   * @var \Drupal\hoeringsportal_base_fixtures\Helper\Helper
+   */
+  protected Helper $baseFixtureHelper;
+
+  /**
+   * Constructor.
+   */
+  public function __construct(Helper $baseFixtureHelper) {
+    $this->baseFixtureHelper = $baseFixtureHelper;
+  }
```

Comment:
In PHP 8.1 this can be written as

```php
  /**
   * Constructor.
   */
  public function __construct(
    readonly private Helper $baseFixtureHelper
  }
```

using Constructor Promotion (https://www.php.net/manual/en/language.oop5.decon.php#language.oop5.decon.constructor.promotion)